content/docs/building-apps/first-deposit.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Fund the Account and make the First Deposit
+order: 65
+---
+
+In this section, we will go over the new user account creation flow between non-custodial wallets and anchors with SEP-24 and/or SEP-6 implementations. Before we dive into the flow, it’s important to understand how Stellar accounts are created in the first place. 
+
+ The first step in creating a Stellar account is to   generate a keypair. This keypair includes public and private keys. However, an account does not exist and does not warrant space on the ledger until it’s  funded with the minimum balance of 1XLM by a Create Account operation. This requirement was created to minimize unused accounts from bloating the ledger. For a more in-depth look at this topic, check out this [page](https://developers.stellar.org/docs/tutorials/create-account/). 
+
+## Who creates the new user accounts? 
+
+When a new customer downloads the wallet application and goes through the deposit flow for the first time, their Stellar account can be created by either the user’s wallet application or the anchor facilitating the first deposit. This section describes the possible strategies and flows for you to consider. 
+
+### Option one: the anchor creates and funds the Stellar account
+
+For this option, the wallet needs to allow users to initiate their first deposit without having to add an asset/establish a trustline. The wallet then prompts the user to add the trustline once funds are received by the anchor. The flow looks like this: 
+
+1. The wallet registers a new user and issues a keypair 
+2. The wallet initiates the first deposit on behalf of the user without requiring the user to add the asset/create the trustline
+3. The anchor provides deposit instructions to the customer 
+4. The user transfers money from a bank account to the anchor’s bank account
+5. Once the anchor receives the transfer, the anchor creates and funds the Stellar account for the customer. *Note*: An anchor should always maintain a healthy amount of XLM in their distribution account to support new account creations. If doing so becomes unsustainable, it’s recommended that the anchor collaborates with wallets to determine a strategy based on the number of account creation requests. The recommended amount is 2XLM per user account creation (1XLM to meet the minimum balance requirement, and 1XLM for establishing trustlines and covering transaction fees).
```

Comment:
I think this note deserves even more emphasis than you give it here, especially since it is relevant in Option 2, as well. Maybe we could remove it from (5) and instead add it into an `<Alert>` block, later? See [here](https://developers.stellar.org/docs/run-api-server/migrating/) for an example of what those look like.

(If you go this route, you'll need to add the following line at the top:

```js
import { Alert } from "components/Alert";
```

after the YAML header.)

Reply:
Either way, I think this deserves to be its own paragraph to highlight its importance!

Reply:
I made it a separate paragraph after step 5!